.editorconfig

```diff
@@ -0,0 +1,621 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = false
+max_line_length = 100
```

Comment:
Please increase the max line length to 150 symbols, 100 is too little and results into multiple line statements.

Reply:
We were thinking about 140. At 150 at least on my laptop it goes off the screen.